src/main/java/codeu/model/store/basic/DefaultDataStore.java

```diff
@@ -24,163 +12,184 @@
 import java.util.List;
 import java.util.UUID;
 
+import org.mindrot.jbcrypt.BCrypt;
```

Comment:
you also need to import codeu.model.data.Mention in between lines 6 and 7 above.